Makefile

```diff
@@ -28,12 +28,12 @@ test: lib
 
 test-syntax:
 	bash ./scripts/test_syntax.sh
-	make reanalyze
+	# make reanalyze
```

Comment:
This not working?

Reply:
I had to deactivate reanalyze for now as there is no 5.x-compatible version of it on opam yet.

Reply:
@cristianoc Ok with you to merge and handle the reanalyze compatibility issue later?

Reply:
Go ahead. Just, one needs to watch out for infinite loops in the meanwhile when the parser is updated.